### PR TITLE
Add a UserAccessToken type + SSH Keys

### PR DESCRIPTION
**Pull request by mmacheerpuppy**
When discussing user access tokens, we refer to them as the parameters token, hash, and expiresAt, which are of type string, string, and time.Time respectively. The usual intention is to avoid exposing the hash and instead enable users to store the token locally, having knowledge of the associated userID and token names. This PR introduces a new type, UserAccessToken (uat), to facilitate conversions from the platformdb type for type safety,  and to maintain consistency throughout the codebase. This is used to help refactor as part of UNW-202.

### CODE REVIEW

**Comment by noorvir** [comment]
Could you also remove the code that is replaced by this PR?

**Comment by mmacheerpuppy** [comment]
> Could you also remove the code that is replaced by this PR?

Deleted all the dead code

File: services/ssh_keys/store.go [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+		PublicKey: pub,
+	})
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func (s Store) GetSSHKeys(ctx context.Context, ownerID string) ([]db.UnweaveSshKey, error) {
```
Comment (noorvir) [inline services/ssh_keys/store.go:39]: Same as above. Should be `SSHKeyAdd` and `SSHKeysList`

File: api/server/routes.go [hunk 2]
```diff
@@ -71,12 +71,6 @@ func API(cfg Config, rti runtime.Initializer, execRouter *router.ExecRouter) {
 		})
 	})
 
-	r.Route("/ssh-keys/{owner}", func(r chi.Router) {
-		r.Post("/", SSHKeyAdd(rti))
-		r.Get("/", SSHKeyList(rti))
-		r.Post("/generate", SSHKeyGenerate(rti))
-	})
-
```
Comment (noorvir) [inline api/server/routes.go:79]: Why were these removed?

Comment (mmacheerpuppy) [reply api/server/routes.go:79]: they are dead, they don't do anything

Comment (noorvir) [reply api/server/routes.go:79]: Aren't they used by the CLI? https://github.com/unweave/cli/blob/master/cmd/keys.go#L85

Comment (mmacheerpuppy) [reply api/server/routes.go:79]: nope, what is used by the CLI is the handler that was in infra, i could have removed this from the master branch and it would be a pure negative diff

Comment (mmacheerpuppy) [reply api/server/routes.go:79]: we should be testing the self-hosted flows in this case

File: services/sshkeys/service.go [hunk 3]
```diff
@@ -16,22 +16,22 @@ import (
...
-func (s *SSHKeyService) Add(ctx context.Context, userID string, params types.SSHKeyAddParams) (string, error) {
+func (s *Service) Add(ctx context.Context, userID string, params types.SSHKeyAddParams) (string, error) {
 	var name string
 	if params.Name != nil && *params.Name != "" {
 		name = *params.Name
 	}
 
 	if name != "" {
-		sshKey, err := s.store.GetSSHKeyByNameIfExists(ctx, name, userID)
+		sshKey, err := s.store.SSHKeyByNameIfExists(ctx, name, userID)
```
Comment (noorvir) [inline services/sshkeys/service.go:34]: Missing "Get" in name => `SSHKeyGetByName`

**Review by noorvir — APPROVED** [review]
